pyproject.toml

```diff
@@ -18,3 +18,42 @@ type = [
   { name = "Improved Documentation",    directory = "doc",     showcontent = true },
   { name = "Trivial Changes",           directory = "trivial", showcontent = false },
 ]
+
+[tool.vendoring]
+destination = "src/pip/_vendor/"
+requirements = "src/pip/_vendor/vendor.txt"
+namespace = "pip._vendor"
+
+protected-files = ["__init__.py", "README.rst", "vendor.txt"]
+patches-dir = "tools/automation/vendoring/patches"
+
+[tool.vendoring.transformations]
+substitute = [
+  # pkg_resource's vendored packages are directly vendored in pip.
+  { match='pkg_resources\.extern', replace='pip._vendor' },
```

Comment:
nitpick

```suggestion
  { match='pkg_resources\.extern', replace="pip._vendor" },
```

Reply:
less nitpicky, use r'...' for the match string, so we're not relying on Python's rule that backslashes before unknown escape characters are retained.

Reply:
This isn't Python but TOML -- single quoted string in TOML are (almost) equivalent to Python's raw strings. :)

Reply:
Oops, sorry. I missed the context.